### PR TITLE
Various q_learning fixes

### PR DESCRIPTION
**Pull request by Nils-Jung**
- Added model save files to gitignore
- Moved launch of q_learning nodes into separate launch file
- Added pytorch dependency to readme 
- Added "use_gpu" argument support to launch file

### CODE REVIEW

File: ros_ws/src/autonomous/q_learning/launch/autonomous_driving.launch [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-<launch>
-<!-- Launches all necessary nodes that let the car drive autonomously. -->
-    <node
-      respawn="true"
-      pkg="q_learning"
-      type="drive.py"
-      name="q_learning_driver"
-      output="screen" >
-    </node>
-</launch>
```
Comment (marian42) [inline ros_ws/src/autonomous/q_learning/launch/autonomous_driving.launch]: Why was this deleted?

File: ros_ws/launch/qlearning.launch [hunk 2]
```diff
@@ -21,19 +22,5 @@
...
-    output="screen" >
-  </node>
-
-  <node
-      pkg="simulation_tools"
-      type="crash_detector"
-      name="crash_detector"
-      output="screen" >
-  </node>
+  <include file="$(find q_learning)/launch/q_learning.launch"/>
```
Comment (marian42) [inline ros_ws/launch/qlearning.launch]: Why did you move these?

File: ros_ws/src/autonomous/q_learning/launch/autonomous_driving.launch (hunk 1 above)
Comment (Nils-Jung) [reply ros_ws/src/autonomous/q_learning/launch/autonomous_driving.launch]: I thought this was a file used for debugging purposes during development that erroneously made it's way onto the master. Sorry, I'll restore it.

File: ros_ws/launch/qlearning.launch (hunk 2 above)
Comment (Nils-Jung) [reply ros_ws/launch/qlearning.launch]: I personally think it is better to have more modular launch files, I think it helps code readability. If you don't agree though or if it messes with your workflow I can revert it of course.
